module-2/backend/week_two.md

```diff
@@ -5,37 +5,201 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. At a high level, what is ActiveRecord? What does it do/allow you to do?
+</br>
+ActiveRecord is a tool that streamlines the CRUD process with built in methods for accomplishing CRUD tasks like create, edit, new, show, delete, and update. 
+</br>
+</br>
+
 2. What is a migration?
+</br>
+When you create a migration, you are identifying what the migration is for (example: Create Students) and then it gives you the class you need to fill in with the information for creating the table of the model. Once you actually migrate, the table should be created in your database with the specified attributes. You can also use a migration for updating an existing table.
+</br>
+</br>
+
 3. How does a table relate to a model?
+</br>
+Every model has a table in the database. The table defines the attributes of the model. For example, if the model is for `student` the table might define attributes like `id`, and `name`.
+</br>
+</br>
+
 4. What kind of methods are `belongs_to`, and `has_many`? (i.e. class or instance) Give an example.
+</br>
+They are class methods. They are used to relate tables to each other. If you have a table for `students` and one for `teachers`, a `teacher` has many students assigned to their table (probably via a `student_id`) and a `student` will belong to only one teacher (in this example). If students actually do have more than one teacher, you'd need a joiner table to relate `students` and `teachers` to each other.
```

Comment:
Tricky question, but these are actually instance methods. The important thing to remember here is that you have to have a specific instance before you can call them. For example:

```ruby
jacob = Student.find(1) # gives us a specific student with an id of 1
mr_fritz = jacob.teacher # uses the `belongs_to :teacher` method on our `student` model to find Jacob's teacher (Jacob being a specific Student instance).
```